src/dashboard/index.ts

```diff
@@ -75,45 +68,51 @@ export default function () {
     })
 
     ipcMain.on('logOut', async (_, message) => {
+      mainWindow.close()
```

Comment:
Is it possible that `mainWindow` could be `null` at runtime here?

Reply:
No because we're running the `registerListeners` function only after window is initialized.